### PR TITLE
Version 0.5.2

### PR DESCRIPTION
**Pull request by samtstern**
Changes:
  * Scopes for Google (#281)
  * Gradle build fix (#272)
  * README colorization for code snippets

@puf please review.

### CODE REVIEW

File: auth/README.md [hunk 1]
```diff
@@ -315,24 +315,37 @@ redefine a string to change it, for example:
 
 ### OAuth Scope Customization
 
+#### Google
+By default, FirebaseUI requests the `email` and `profile` scopes when using Google Sign In. If you
+would like to request additional scopes from the user, add a string array resource to your
+application like this:
```
Comment (puf) [inline auth/README.md]: Where do I add this? A new element in an existing file? Or is it a new file? If so, does the name of the file matter? Or is it only about the array's name?

File: auth/README.md [hunk 2]
```diff
@@ -315,24 +315,37 @@ redefine a string to change it, for example:
...
+    <item>https://www.googleapis.com/auth/drive.readonly</item>
+</array>
+```
+
+
 #### Facebook
 
 By default, FirebaseUI requests the `email` and `public_profile` permissions when initiating
 Facebook Login.  If you would like to override these scopes, add a string array resource
 to your application like this:
```
Comment (puf) [inline auth/README.md]: Same as before: where does this XML go?

**Comment by puf** [comment]
A few comments. Aside from that lgtm.

**Comment by samtstern** [comment]
@puf clarified XML setup

**Pull request merged**